android/app/build.gradle

```diff
@@ -239,3 +267,6 @@ tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).all {
         jvmTarget = "1.8"
     }
 }
+
+// Necessary for Autolinking; see facebook/react-native@261197d85
+apply from: file("../../node_modules/@react-native-community/cli-platform-android/native_modules.gradle"); applyNativeModulesAppBuildGradle(project)
```

Comment:
nit: newline at end of line

Also, let's uncramp this line by putting the statements on separate lines. I can see why people jam them into one line when writing instructions and saying to copy-paste; but now that it's part of our source code, it's better to put the code in a reasonable format.